### PR TITLE
Update spanish.json

### PR DESCRIPTION
**Pull request by MauricioLanda**
typo at line 709: serveral instead of several

### CODE REVIEW

File: i18n/spanish.json [hunk 1]
```diff
@@ -702,34 +702,34 @@
...
+    "cmd-blindtest-help" : "Inicia una prueba a ciegas en el canal actual.\n\nOpciones Disponibles:\n\nlongestword=true|false (Default to False):\nAceptar la palabra más larga como una buena respuesta.\n\nlistening=true|false (Default to False):\n No acepta respuestas, solo escucha las canciones.\n\ntimeout=number (Default to 120):\n Tiempo de espera antes de detener la canción si no se encuentra. Entre 3 y 300 segundos.\n\nseverity=number (Default to 100):\n Severidad en porcentaje de palabra para aceptar una respuesta. Entre 1 y 100%.\n\npercentage=number,number,number (Default to 100,0,0):\n Porcentaje de OP, ED, OST. La suma de todos los valores debe ser 100.\n\nwait=number (Default to 5):\n Tiempo para esperar antes de cada canción. Entre 0 y 30 segundos.\n\nargument=status (Default to all):\n Solo selecciona un tipo de anime visto en las listas de anime dadas.\nLos estados disponibles son: todos, completado, observación, retener, descartado, ptw.\n\nremove=status,status2 (Default to ptw):\n Ignora un tipo (o varios) de anime visto (separados por comas) en las listas de anime dadas.\nLos estados disponibles son: completado, observación, retención, descartado, ptw.",
```
Comment (Zenrac) [inline i18n/spanish.json]: Do not translate the available status as they have to be written in english only

**Comment by MauricioLanda** [comment]
Ok, i will fix that in a few hours and let you know with another PR

**Comment by Zenrac** [comment]
do not open another pull request, just edit your file in your fork, it will auto-update this pull request.

**Comment by MauricioLanda** [comment]
Very well, thank you

**Comment by MauricioLanda** [comment]
it is now changed

**Comment by Zenrac** [comment]
Thanks

**Pull request merged**